src/frontend/js/components/AddressesManagement/AddressForm.tsx

```diff
@@ -41,6 +42,22 @@ const AddressForm = ({ handleReset, onSubmit, address }: Props) => {
   const [languageCode] = intl.locale.split('-');
   const countryList = countries.getNames(languageCode);
 
+  const getLocalizedErrorMessage = (
+    error: Maybe<
+      | string
+      | {
+          key: ErrorKeys;
+          values: Record<PropertyKey, string | number | Array<string | number>>;
+        }
+    >,
+  ) => {
+    if (!error) return undefined;
+    if (typeof error === 'string') return error;
```

Comment:
If it's directly an error you don't want to translate it? it should still be a key or it's only for the default errors of yup?

Reply:
It is just in case, the error has not been customized, I would like to be able to conserve the default behaviour.